src/inc/corjit.h

```diff
@@ -177,6 +177,8 @@ enum CorJitFlag2
     CORJIT_FLG2_SAMPLING_JIT_BACKGROUND = 0x00000001, // JIT is being invoked as a result of stack sampling for hot methods in the background
 #if COR_JIT_EE_VERSION > 460
     CORJIT_FLG2_USE_PINVOKE_HELPERS     = 0x00000002, // The JIT should use the PINVOKE_{BEGIN,END} helpers instead of emitting inline transitions
+    CORJIT_FLG2_ENABLE_INTRINSIC        = 0x00000004, // The JIT emits intrinsic as possible.
+    CORJIT_FLG2_DISABLE_INTRINSIC       = 0x00000008, // The JIT emits call instead of intrinsic.
```

Comment:
Of course, we can't set both in theory, but I make it default to DISABLE in such case. Ideally I want a flag that can be set in either direction. Unfortunately, the current flag only allows a boolean value so I had to use two. For instance, under MIN_OPT, without ENABLE flag, I cannot turn on intrinsic expansion unless I want to change the current default behavior.